modules/apps/frontend-taglib/frontend-taglib-clay/package.json

```diff
@@ -82,11 +82,13 @@
 		"react-dom": "16.12.0"
 	},
 	"name": "frontend-taglib-clay",
+	"main": "index.js",
```

Comment:
I'm curious about this change, was this required for something specific? Or was this primarily because you added the index.js file?

Reply:
Good question, and this is still a bit of a "draft": the goal is to be able to import components that are exported from `index.js` in any other module. In my case here I'm just using it in `portal-web`, because I want to render the `TranslationSelector` from the [`InputLocalized`](https://github.com/liferay/liferay-portal/blob/167f47e46c2ea38101a1cfbf6d2ba237e138b484/portal-web/docroot/html/taglib/ui/input_localized/page.jsp) tag, but in the long run we'd also want for devs to be able to just render these "translation" components in their React app too.